samples/maps/chart/events-click/demo.js

```diff
@@ -50,10 +50,11 @@ Highcharts.getJSON('https://cdn.jsdelivr.net/gh/highcharts/highcharts@v7.0.0/sam
                 valueSuffix: '/km²'
             }
         }, {
+            colorAxis: false,
```

Comment:
@TorsteinHonsi We should take the chance and transition to
```
colorAxis: {
    enabled: false
}
```

Reply:
Just to be on the same page, I prefer one type per option instead of `undefined/string/number/boolean`. All strongly typed wrappers (e.g. iOS) can't create one-to-one option for multi-typed options. So something like `colorAxis: { enabled, index, id }` would be the best for those wrappers :)

With the config above, we have similar settings for all: `series.xAxis`, `series.yAxis` and `series.colorAxis` (except extra boolean for disabling colorAxis - which makes sense, you shouldn't be able to disable xAxis in cartesian series). Leveraging of the same config, we can use `bindAxes()` for colorAxis without any hacks.

Reply:
I see. So the naming is wrong. It does not take any options, but is a simple flag.

Reply:
It's `string/number/boolean/undefined` so quite a flag I would say 😅